### PR TITLE
Ima

### PR DESCRIPTION
**Pull request by wtao-mux**

### CODE REVIEW

File: MUXSDKStats/Podfile [hunk 1]
```diff
@@ -1,12 +1,13 @@
 # Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```
Comment (jsanford8) [inline MUXSDKStats/Podfile]: Is this going to force us to iOS 9+? And if so, is that what the IMA SDK requires?

File: MUXSDKStats/Podfile [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 target 'MUXSDKStats' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   # use_frameworks!
 
   # Pods for MUXSDKStats
   pod 'Mux-Stats-Core', '~>2.0.0'
+  pod 'GoogleAds-IMA-iOS-SDK', '~> 3.7'
```
Comment (jsanford8) [inline MUXSDKStats/Podfile]: How do podfiles work? For instance, if you now load our pod, does that automatically include the IMA Pod in your project? Or is that stripped out if you don't add that yourself explicitly?

File: MUXSDKStats/Podfile (hunk 1 above)
Comment (wtao-mux) [reply MUXSDKStats/Podfile]: the IMA SDK has a minimum iOS9+ requirement. that's why the uncomment

File: MUXSDKStats/Podfile (hunk 2 above)
Comment (wtao-mux) [reply MUXSDKStats/Podfile]: no. framework don't embed framework. open up, the DemoApp's Podfile, you will see all three pods are mentioned

File: MUXSDKStats/Podfile (hunk 1 above)
Comment (jsanford8) [reply MUXSDKStats/Podfile]: ahh interesting, okay

File: MUXSDKStats/Podfile (hunk 2 above)
Comment (jsanford8) [reply MUXSDKStats/Podfile]: nice 👍

**Pull request merged**